src/screens/tabs/Profile/modals/ImportKey/index.tsx

```diff
@@ -75,9 +82,9 @@ function ImportKey({ createImportRef, modalRef, accountsModalRef }: Props) {
           style={styles.inputStyle}
           onChangeText={handleOnChangeKey}
         />
-        {error && (
+        {errorType && (
           <Text type="caption" style={styles.error}>
-            {t('createImportAccount.invalidKey')}
+            {t(`createImportAccount.${errorType}`)}
```

Comment:
instead of camel casing the type, and do this here, what if we create a function in utils with a switch that returns an error for each type? So we can avoid the camel case logic and put default error in case backend changes/adds a new message